src/Toolkit/kits/shadcn/AlertDialog/templates/components/AlertDialog/Content.html.twig

```diff
@@ -0,0 +1,37 @@
+<dialog
+    id="{{ _alert_dialog_id }}"
+    {{ _alert_dialog_open ? 'open' }}
+    class="{{ 'fixed inset-0 size-auto max-h-none max-w-none overflow-y-auto bg-transparent backdrop:bg-transparent ' ~ attributes.render('class')|tailwind_merge }}"
+    data-alert-dialog-target="dialog"
+    data-action="keydown.esc->alert-dialog#close:prevent"
+    {{ attributes.without('id') }}
+>
+    <div
+        data-alert-dialog-target="overlay"
+        data-transition-enter="transition ease-out duration-100"
+        data-transition-enter-start="transform opacity-0"
+        data-transition-enter-end="transform opacity-100"
+        data-transition-leave="transition ease-in duration-75"
+        data-transition-leave-start="transform opacity-100"
+        data-transition-leave-end="transform opacity-0"
```

Comment:
This `data-transition-*` syntax from Vue/Alpine.js is do-able thanks to https://github.com/mmccall10/el-transition

FYI it's not possible to use https://stimulus-use.github.io/stimulus-use/#/use-transition because it supports only one target, but we need to transition the overlay and content targets.